cpu/k60/include/MK60D10.h

```diff
@@ -73,7 +73,7 @@
 */
 
 /*!
- * @file MK60D10.h
+ * @file
```

Comment:
I keep forgetting the details - didn't we want to leave these CMSIS files unchanged (except for trailing whitespace)?